www/unstable/userguide/weekyears.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: userguide
+title: Week years
+category: advanced
+weight: 3037
+---
+
+Introducing week-years
+===
+
+Most of the time, dates are represented as a year, month and day in
+a particular calendar system - occasionally with an era as well. An
+alternative approach is to represent a date as a week-year, week,
+and day-of-week. This is performed relative to a specific calendar
+system, along with a rule used to work out when each week-year starts.
+
+For example, using the ISO-8601 rule, the year/month/day combination
+"2016, February, 4" is equivalently week-year 2016, week 5,
```

Comment:
In fact, if you move the "This is performed relative" line above to below this para, you could use "year" here again, and then follow with "However, the year/month/day combination "2016, January, 1" falls into year 2015, week 53 [TODO: substitute actual valid example], and so we call this type of year a "week-year".

Or similar.